manim/scene/scene.py

```diff
@@ -980,18 +980,31 @@ def update_to_time(self, t):
     def add_sound(self, sound_file, time_offset=0, gain=None, **kwargs):
         """
         This method is used to add a sound to the animation.
-
-        Parameters
-        ----------
-        sound_file : str
-            The path to the sound file.
-
-        time_offset : int,float, optional
             The offset in the sound file after which
             the sound can be played.
 
-        gain :
+        gain : float
+            Amplification of the sound.
```

Comment:
There is something off in the diff here, please check whether things look correct.

Reply:
thanks, I updated this